plugin/webhook/webhook.go

```diff
@@ -34,13 +34,32 @@ const (
 	WebhookError Level = "ERROR"
 )
 
+// Issue object of issue
+type Issue struct {
+	ID          int    `json:"id"`
+	Name        string `json:"name"`
+	Status      string `json:"status"`
+	Type        string `json:"type"`
+	Description string `json:"description"`
+}
+
+// Project object of project
+type Project struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
 // Context is the context of webhook.
 type Context struct {
 	URL          string
+	ActivityType string
+	Issue        Issue
+	Project      Project
 	Level        Level
 	Title        string
 	Description  string
 	Link         string
+	CreatorID    int
 	CreatorName  string
 	CreatorEmail string
```

Comment:
As a 2nd thought, let's not include email as it's the PII info and may introduce data privacy concerns.

We can re-evaluate later if the email info is badly needed.

Reply:
CreatorEmail removed.

Reply:
CreatorEmail removed.